src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -749,3 +749,10 @@ COOP_PINVOKE_HELPER(void, RhGetCurrentThreadStackBounds, (PTR_VOID * ppStackLow,
 {
     ThreadStore::GetCurrentThread()->GetStackBounds(ppStackLow, ppStackHigh);
 }
+
+ThreadExitCallback g_threadExitCallback;
```

Comment:
I moved the comment describing g_threadExitCallback to the file where it is defined.